src/main/java/core/basesyntax/model/Activity.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Activity {
```

Comment:
let's call this `FruitTransaction`
current name is very confusing

Reply:
)